lib/matplotlib/gridspec.py

```diff
@@ -320,7 +321,12 @@ def __setstate__(self, state):
 
     def update(self, **kwargs):
         """
-        Update the current values.
+        Update the subplot parameters of the grid.
```

Comment:
I'm more specific *which* values to update.

There's also some position updating of some Axes happening, which I don't understand, so I'm still not describing that.